ansible_rulebook/cli.py

```diff
@@ -136,6 +137,12 @@ def get_parser() -> argparse.ArgumentParser:
         "graceful shutdown, default: 60. The process will shutdown if "
         "all actions complete before this time period",
     )
+    parser.add_argument(
+        "--gc-after",
+        default=os.environ.get("EDA_GC_AFTER", DEFAULT_GC_AFTER),
+        type=int,
+        help="Run the garbage collector after this number of events",
```

Comment:
```suggestion
        help="Run the garbage collector after this number of events."
        "It can be configured with the environment variable EDA_GC_AFTER",
```